node/sc/vt_recovery.go

```diff
@@ -244,8 +244,8 @@ func (vtr *valueTransferRecovery) retrievePendingEvents() error {
 
 // retrievePendingEventsFrom retrieves pending events from the specified bridge by using the hint provided.
 // The filter uses a hint as a search range. It returns a slice of events that has log details.
-func retrievePendingEventsFrom(hint *valueTransferHint, bi *BridgeInfo) ([]*RequestValueTransferEvent, error) {
-	if bi.bridge == nil {
+func retrievePendingEventsFrom(hint *valueTransferHint, from, to *BridgeInfo) ([]*RequestValueTransferEvent, error) {
+	if from.bridge == nil {
```

Comment:
Don't we need to check if `to.bridge` is nil or not?

Reply:
Good point. I reflected.